Add tests to travis

slsh1o opened the pull request.

slsh1o committed:
Add job to run tests & matrix to travis

webknjaz reviewed (changes requested):
It will have just one job unless you add some matrix expansion:
https://docs.travis-ci.com/user/build-matrix/#explicitly-included-jobs-with-only-one-element-in-the-build-matrix

Also, the PR title shouldn't end with a period. Plus the commit message should contain the summary, not repeat each and every line of the diff. Implementation details shouldn't be in the commit title — the commit itself already contains the implementation that is perfectly readable. What is the high-level effect of this commit? That's what the commit message title should answer.

slsh1o commented:
Ok now travis able to run tests but failed with empty secret key. Looks like I should implement #27 first.

webknjaz commented on .travis.yml:
```diff
@@ -1,5 +1,12 @@
-python: 3.8
+python:
+  - 3.8
+  - 3.8
```
Don't use the same version twice

webknjaz reviewed (changes requested):
You may use https://docs.travis-ci.com/user/environment-variables/#defining-public-variables-in-travisyml to define some env vars for now.

slsh1o committed:
Add dummy secret key for travis build

Django app will fail without SECRET_KEY var so dummy key was added for
travis builds.
